lib/slim_fast/parser.ex

```diff
@@ -101,7 +101,7 @@ defmodule SlimFast.Parser do
              |> String.split(~r/^(-|==|=)/)
              |> List.last
              |> String.lstrip
-    inline = inline or String.starts_with?(input, "=")
+    inline = inline or String.starts_with?(input, "=") or String.starts_with?(script, "if")
```

Comment:
Thanks for the reply @henrik, sorry for taking so long to get back to you.  You're right about bounding the `if` and the additional expressions.

As for support, I do think we will encounter situations where Slim/Ruby and EEx/Elixir don't see eye to eye.  If I'm not mistaken that was the case with the `\n` I needed to include to support `case` and `cond` statements.  

I'd like to get as close to complete Ruby Slim support while staying true to Elixir.  What are your thoughts on a direction here?  /cc @Rakoth